numpy/lib/tests/test_io.py

```diff
@@ -637,6 +637,37 @@ def check_large_zip(memoryerror_raised):
             pytest.xfail("subprocess got a SIGKILL, apparently free memory was not sufficient")
         assert p.exitcode == 0
 
+    def test_fstring_fmt(self):
```

Comment:
You added support for complex number formatting with `fstring_fmt`. Can you add tests for that case as well?